README.md

```diff
@@ -1 +1,79 @@
-# react-capstone-project
\ No newline at end of file
+# react-capstone-project
+In this project you can check for some nintendo characters, has a detail page for each character, this is a Single Page App Built with React, JavaScript and HTML,  it has been tested so you can also look for the testing files.
+
+## Built With:
+
+- Major languages: Javascript, HTML, CSS
+- Frameworks: React, Redux, Babel, CRA
+- Technologies used: VSC, Github
+
+## APIs used for this project : 
+
+- https://www.amiiboapi.com/
+
+## Live Demo (if available)
+
+- See live demo: https://maury-react-capstone.herokuapp.com/
+
```

Comment:
- [ ] I would like to recommend you put your demo video link here in your readme file so that other people will be able to see your presentation and that will attract them :+1: